### DISCUSSION TITLE
这是怎么做到的？油管看不了google能用……

### DISCUSSION THREAD

**Discussion by daiaji** [v2fly/v2ray-core#714] [Q&A]
https://github.com/v2fly/v2ray-core/issues/713
方案就是cloudflare+caddy+v2ray
感觉就是中途连接就被干掉了。
服务端那边根本没有任何提示。
但google似乎还能用……
SNI也不至于吧
真有有明文的youtube字符串吗？

**Comment by Loyalsoldier** [comment]


**Reply by daiaji** [reply]
curl也这样……
而且我在用firefox
